mkdocs/tests/config/config_options_tests.py

```diff
@@ -280,39 +280,34 @@ def test_invalid_IPv6_address(self):
 class URLTest(unittest.TestCase):
 
     def test_valid_url(self):
+        option = config_options.URL()
 
-        url = "https://mkdocs.org"
+        self.assertEqual(option.validate("https://mkdocs.org"), "https://mkdocs.org")
+        self.assertEqual(option.validate(""), "")
 
-        option = config_options.URL()
-        value = option.validate(url)
-        self.assertEqual(value, url)
+    def test_valid_url_is_dir(self):
+        option = config_options.URL(is_dir=True)
 
-    def test_invalid_url(self):
+        self.assertEqual(option.validate("http://mkdocs.org/"), "http://mkdocs.org/")
+        self.assertEqual(option.validate("https://mkdocs.org"), "https://mkdocs.org/")
 
+    def test_invalid_url(self):
         option = config_options.URL()
+
         self.assertRaises(config_options.ValidationError,
                           option.validate, "www.mkdocs.org")
+        self.assertRaises(config_options.ValidationError,
+                          option.validate, "//mkdocs.org/test")
+        self.assertRaises(config_options.ValidationError,
+                          option.validate, "http:/mkdocs.org/")
+        self.assertRaises(config_options.ValidationError,
+                          option.validate, "/hello/")
 
-    def test_invalid(self):
-
+    def test_invalid_type(self):
         option = config_options.URL()
         self.assertRaises(config_options.ValidationError,
                           option.validate, 1)
 
-    def test_url_is_dir(self):
-        url = "https://mkdocs.org/"
-
-        option = config_options.URL(is_dir=True)
-        value = option.validate(url)
-        self.assertEqual(value, url)
-
-    def test_url_transform_to_dir(self):
```

Comment:
It looks like what this test was covering is not checked anymore if you remove that test or I might be missing something?

Reply:
The line `self.assertEqual(option.validate("https://mkdocs.org"), "https://mkdocs.org/")` covers exactly this